cmake/find/Findlibbacktrace.cmake

```diff
@@ -0,0 +1,70 @@
+# Distributed under the OSI-approved BSD 3-Clause License.  See accompanying
+# file Copyright.txt or https://cmake.org/licensing for details.
+
+#.rst:
+# Findlibbacktrace
+# -------
+#
+# Finds the libbacktrace library
+#
+# This will define the following variables::
+#
+#   libbacktrace_FOUND    - True if the system has the libbacktrace library
+#   libbacktrace_VERSION  - The version of the libbacktrace library which was found
+#
+# and the following imported targets::
+#
+#   libbacktrace::libbacktrace   - The libbacktrace library
+
+find_path(libbacktrace_INCLUDE_DIR
+  NAMES backtrace.h
+  PATHS "${libbacktrace_ROOT}/include"
+)
+
+if(WIN32)
+  set(_libbacktrace_ORIG_CMAKE_FIND_LIBRARY_SUFFIXES ${CMAKE_FIND_LIBRARY_SUFFIXES})
+  set(_libbacktrace_ORIG_CMAKE_FIND_LIBRARY_PREFIXES ${CMAKE_FIND_LIBRARY_PREFIXES})
+  set(CMAKE_FIND_LIBRARY_SUFFIXES .so .a ${CMAKE_FIND_LIBRARY_SUFFIXES})
+  set(CMAKE_FIND_LIBRARY_PREFIXES lib ${CMAKE_FIND_LIBRARY_PREFIXES})
+endif()
+
+find_library(libbacktrace_LIBRARY
+  NAMES backtrace
+  PATHS "${libbacktrace_ROOT}/lib"
+)
+
+if(WIN32)
+  set(CMAKE_FIND_LIBRARY_SUFFIXES ${_libbacktrace_ORIG_CMAKE_FIND_LIBRARY_SUFFIXES})
```

Comment:
Why is the variable reset again. Could you explain?

Reply:
It should be "reset" because other `find_library()` in other modules etc. shouldn't be affected ! See also FindBoost and other find modules. They all do it that way ... I added a comment.